doc/modules/linear_model.rst

```diff
@@ -39,14 +39,16 @@ solves a problem of the form:
 
 :class:`LinearRegression` will take in its ``fit`` method arrays X, y
 and will store the coefficients :math:`w` of the linear model in its
-``coef_`` member::
+``coef_`` and ``intercept_`` members::
 
     >>> from sklearn import linear_model
     >>> reg = linear_model.LinearRegression()
     >>> reg.fit([[0, 0], [1, 1], [2, 2]], [0, 1, 2])
     LinearRegression()
     >>> reg.coef_
     array([0.5, 0.5])
+    >>> reg.intercept_
+    1.1102230246251565e-16
```

Comment:
I think we can also adjust the example a little bit to have a more interesting `intercept_`:

```rst
    >>> from sklearn import linear_model
    >>> reg = linear_model.LinearRegression()
    >>> reg.fit([[0, 0], [1, 1], [2, 2]], [1, 2, 3])
    LinearRegression()
    >>> reg.coef_
    array([0.5, 0.5])
    >>> reg.intercept_
    1.0
```

Reply:
Good idea. Done.